### ISSUE TITLE
Issue with Bands_Period_M15/Bands_Period_M30

### ISSUE THREAD

**Issue by kenorb** [EA31337/EA31337-Lite-optimization-tests#9]
```
         Pass  Profit Total trades Profit factor Expected Payoff Drawdown $ Drawdown %
            1 -441.45          495          0.89           -0.89     673.58       0.67  Bands_Period_M15=2; MaxOrders=0; MaxOrdersPerType=0;
            2 -441.45          495          0.89           -0.89     673.58       0.67  Bands_Period_M15=4; MaxOrders=0; MaxOrdersPerType=0;
            3 -441.45          495          0.89           -0.89     673.58       0.67  Bands_Period_M15=6; MaxOrders=0; MaxOrdersPerType=0;
            4 -441.45          495          0.89           -0.89     673.58       0.67  Bands_Period_M15=8; MaxOrders=0; MaxOrdersPerType=0;
            5 -441.45          495          0.89           -0.89     673.58       0.67 Bands_Period_M15=10; MaxOrders=0; MaxOrdersPerType=0;
            6 -441.45          495          0.89           -0.89     673.58       0.67 Bands_Period_M15=12; MaxOrders=0; MaxOrdersPerType=0;
            7 -441.45          495          0.89           -0.89     673.58       0.67 Bands_Period_M15=14; MaxOrders=0; MaxOrdersPerType=0;
            8 -441.45          495          0.89           -0.89     673.58       0.67 Bands_Period_M15=16; MaxOrders=0; MaxOrdersPerType=0;
            9 -441.45          495          0.89           -0.89     673.58       0.67 Bands_Period_M15=18; MaxOrders=0; MaxOrdersPerType=0;
```

```
          Pass Profit Total trades Profit factor Expected Payoff Drawdown $ Drawdown %
             1 -26.26          517          1.00           -0.05     803.27       0.80  Bands_Period_M30=2; MaxOrders=0; MaxOrdersPerType=0;
             2 -26.26          517          1.00           -0.05     803.27       0.80  Bands_Period_M30=4; MaxOrders=0; MaxOrdersPerType=0;
             3 -26.26          517          1.00           -0.05     803.27       0.80  Bands_Period_M30=6; MaxOrders=0; MaxOrdersPerType=0;
             4 -26.26          517          1.00           -0.05     803.27       0.80  Bands_Period_M30=8; MaxOrders=0; MaxOrdersPerType=0;
             5 -26.26          517          1.00           -0.05     803.27       0.80 Bands_Period_M30=10; MaxOrders=0; MaxOrdersPerType=0;
```

Build: [799661990](https://github.com/EA31337/EA31337-Lite-optimization-tests/runs/799661990?check_suite_focus=true#step:5:269)

**Comment by kenorb** [comment]
Cause: negative Bands_Deviation.

**Comment by kenorb** [comment]
Fixed in #10
